extensions/amp-consent/0.1/consent-config.js

```diff
@@ -250,14 +249,18 @@ export class ConsentConfig {
    * @return {!JsonObject}
    */
   checkStoryConsent_(config) {
-    if (childElementByTag(this.element_, AMP_STORY_CONSENT_TAG)) {
-      userAssert(
-        !config['promptUISrc'],
-        '%s: `promptUiSrc` cannot be specified while using %s.',
-        TAG,
-        AMP_STORY_CONSENT_TAG
-      );
-    }
+    // TODO(wg-stories):
+    // We verify that configurations with an <amp-story-consent> descendant have
+    // no promptUiSrc, but we don't verify that the element is present in
+    // Stories with <amp-cosent>. Thus, Stories could be configured using a
+    // promptUiSrc. This is likely not what we want for UX, so wg-stories
+    // should provide a migration path for these documents if they exist.
+    userAssert(
+      !config['promptUISrc'] ||
+        !childElementByTag(this.element_, 'amp-story-consent'),
```

Comment:
Flipped condition to avoid an unnecessary querySelection

Reply:
```suggestion
      !(config['promptUISrc'] &&
        childElementByTag(this.element_, 'amp-story-consent')),
```